scripts/modules/cli.py

```diff
@@ -565,7 +576,15 @@ def start_handler(self):
             ),
         )
         docker_compose_path = args["docker_compose_path"]
-        json.dump(compose, docker_compose_path, indent=2, sort_keys=True)
+
+        if args.get("output_format") == 'yaml':
+            yaml.dump(compose, docker_compose_path,
+                      explicit_start=True,
+                      default_flow_style=False,
+                      indent=2
+                      )
+        else:
```

Comment:
Nit: This block will be entered if anything other than `yaml` is specified or no option is specified and the default is used. This may include misspellings such as `yamll`, for example, which should throw an error instead of silently switching to `json`, IMHO. Perhaps replace with an `elif`?